tests/explorer.api.tests/ExploreTests.cs

```diff
@@ -190,7 +190,7 @@ public void FailWithMissingFields()
         {
             // TestUtils.WaitDebugger();
             using var client = factory.CreateExplorerApiHttpClient(nameof(ExploreTests), vcrSessionName);
-            using var request = factory.CreateHttpRequest(method, endpoint, data);
+            using var request = factory.CreateHttpRequest(method, endpoint, data!);
```

Comment:
I had to look up what the `!` does here... I haven't come across it before. 

My understanding is that it's way to signal to the compiler "don't worry, this is never null" - which isn't the case. I think it would be more appropriate to add the `?` modifier to the receiving method signature to signal that the method accepts and handles null references.